src/crimson/net/SocketConnection.h

```diff
@@ -40,6 +40,13 @@ class SocketConnection : public Connection {
   Dispatcher& dispatcher;
   seastar::gate pending_dispatch;
 
+  /// my_socket_port can be different from my_addr.get_port() if is
+  /// connector.
+  int my_socket_port = 0;
```

Comment:
i'd suggest use `uint16_t` for port number. we could fix `entity_addr_t` side later on.

Reply:
this comment is not addressed.

Reply:
fixed